common/src/main/java/com/adaptivescale/rosetta/common/types/RosettaModuleTypes.java

```diff
@@ -24,4 +24,5 @@ public enum RosettaModuleTypes {
     COLUMN_EXTRACTOR,
     TABLE_EXTRACTOR,
     VIEW_EXTRACTOR,
+    DIFF_TESTER,
```

Comment:
For the **DIFF_TESTER** we had only the DefaultTester(). Since for Kinetica we don't have Foreign Key support, the fix required to create a KineticaTester module